### ISSUE TITLE
Please help, AJAX blank popups

### ISSUE THREAD

**Issue by palentir** [osTicket/osTicket#3261]
Hi, I've read a bit about my issue, seems AJAX isnt working? When in OSticket admin section and click resolve, the popup window is small and empty. I wonder if anyone can help me? I have a domain which uses subfolders (so I can use other domains and point them to the same ftp), so for example, www.mydomain.com forwarding to \html\mydomain\..
So, when I went to my site the URL looked like mydomain.com\mydomain\
So I used htaccess to hide the mydomain subfolder from the url, I think this is related to my issue.

So my first problem was the osticket not working because the path was messed up. I had to add this to the top of my bootstrap.php to fix it:

define('ROOT_PATH', 'http://mydomain.com/support/');

This worked nice, but now the admin section is not showing the popups. I am considering this could be related to the path still? Can anyone recommend a possible fix, similar to the one I used for the customer portal section?

my htaccess file:
RewriteCond %{HTTP_HOST} mydomain.com
RewriteCond %{REQUEST_URI} !^/mydomain
RewriteRule ^(.*)$ mydomain/$1 [L]

![b5bf6a5e-471b-11e5-880e-10ea7e949e98](https://cloud.githubusercontent.com/assets/20346741/16673288/cd6c230e-44a4-11e6-9c12-4a64767809fb.png)

**Comment by kest874** [comment]
Might check your error.log and see if there is a php error, will help you to troubleshoot.

**Comment by palentir** [comment]
@kest874 Thanks for the reply, I use a web hosting console so I dont really have access to Apache. What would you suggest in such a situation?

**Comment by ntozier** [comment]
I would think that this type of question would be better suited for the support forums. 
http://osticket.com/forum

When requesting help you should always include the version of osTicket that you are running, Webserver software, PHP version, and error logs.  There are also a number of threads both on the forums and here on github that discuss the AJAX issue(s).  

Every web host I have ever worked with has allowed access to web / PHP logs.  If you are using a el cheapo hosting that doesn't then I would suggest that you find a new host.

**Comment by palentir** [comment]
@ntozier thats for the suggestion, I will post there and include the recommended information.

I spoke to my webhost, they said there is a button in my console that lets me download the error log. But the zip file is empty and its also empty in the boxes on my console.

**Comment by palentir** [comment]
Hi, I have resolved my issue. I had to restructure my hosting to force this to work. So the website which I am using osTicket with... I had to remove its subfolder and put that website's php files in the /public_html/ folder root. All my other sites use a rewrite rule, whereas this main site just sits in the root. It looks ugly, is quite annoying, but the osticket now works and the end users do not see my messy folder structures. So I guess it is a good outcome. Thanks again for help so far. Maybe my workaround will help someone else out there with similar troubles.
